packages/php/email-editor/src/Engine/class-personalizer.php

```diff
@@ -122,6 +122,25 @@ public function personalize_content( string $content ): string {
 					$content_processor->remove_attribute( 'data-link-href' );
 					$content_processor->remove_attribute( 'contenteditable' );
 				}
+			} elseif ( $content_processor->get_token_type() === '#tag' && $content_processor->get_tag() === 'A' ) {
+				$href = $content_processor->get_attribute( 'href' );
+
+				if ( ! $href || ! preg_match( '/\[[a-z-\/]+\]/', urldecode( $href ), $matches ) ) {
```

Comment:
I am not sure about the syntax. We used the HTML comments, but here we are using only the syntax `[]`. Should we be more consistent and use the html comments here as well?

Reply:
On discussing the ticket, we decided to use the syntax without the comment part in this specific case, because it looks less confusing when displayed in the link control, and it's similar to the shortcode syntax that users might be more familiar with.